engine/ecs/private/components/point_light_component.cpp

```diff
@@ -0,0 +1,18 @@
+#include "components/point_light_component.hpp"
+#include "glm/gtc/type_ptr.hpp"
+
+namespace EnttEditor
+{
+template <>
+void ComponentEditorWidget<PointLightComponent>(entt::registry& reg, entt::registry::entity_type e)
+{
+    auto& t = reg.get<PointLightComponent>(e);
+    t.Inspect();
+}
+}
+void PointLightComponent::Inspect()
```

Comment:
So do I just remove it then? Because I thought being able to edit the pointlight data during runtime would be good. Or do you want to move this functionality somewhere else? I am not familiar with the Editor things yet